internal/resourcecache/resourcecache.go

```diff
@@ -0,0 +1,109 @@
+package resourcecache
+
+import (
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var sleepTimeBeforeCleanup = 2 * time.Minute
```

Comment:
Do want to make this configurable?

Reply:
:shrug: doesn't seem like a tunable we *need* and our config is already pretty big. I'm not that inclined to add it

Reply:
`s/var/const/`?

Reply:
good call